apps/resources/admin.py

```diff
@@ -80,7 +86,35 @@ def get_list_filter(self, request):
         Returns a sequence containing the fields to be displayed as filters in
         the right sidebar of the changelist page.
         """
-        list_filter = ['status', 'tags']
+        list_filter = ['status', 'categories', 'tags']
         if request.user.is_superuser:
             list_filter.append('organisation')
+
         return list_filter
+
+    def categories_list(self, obj):
+        return ', '.join(obj.categories.all().values_list('title', flat=True))
+
+    categories_list.short_description = 'Categories'
+
+    def action_categorise(self, request, queryset):
+        template = 'resources/admin/categorise_resources.html'
+
+        ids = ','.join(str(pk) for pk in queryset.values_list('pk', flat=True))
+        categories = models.ResourceCategory.objects.all()
+
+        messages.info(request, '{} resources selected'.format(queryset.count()))
+
+        return render(
+            request, template, {
+                "title": "Categorise Resources",
+                "site_title": admin.site.site_title,
+                "site_header": admin.site.site_header,
+                "categories": categories,
+                "resource_ids": ids,
+                "post_to": reverse('resources:admin-categorise-resources'),
+            }
+        )
+
+    action_categorise.short_description = 'Categorise selected resources'
```

Comment:
is it worth renaming this to `Recategorise Selected Resources` as categories can be removed as well as added?

Reply:
"Add/Remove resources to/from Categories"?
Yeah, I couldn't make a nice phrase...